src/baldr/graphreader.cc

```diff
@@ -390,8 +413,12 @@ const GraphTile* GraphReader::GetGraphTile(const GraphId& graphid) {
       return nullptr;
     }
 
+    auto traffic_ptr = tile_extract_->traffic_tiles.find(base);
```

Comment:
in the future we will probably need to decouple the idea of flat file and memory mapped for graph and traffic tiles. it would be nice to fall back to flat files for traffic tiles as well and then to fall back to network calls etc. but all that can come once we figure something out for cache invalidation for traffic tiles from file vs memmap. i just wanted to point out the future work here is all, not asking for changes